executor/analyze_test.go

```diff
@@ -123,7 +123,7 @@ func (s *testSuite1) TestAnalyzeReplicaReadFollower(c *C) {
 	tk.MustExec("analyze table t")
 }
 
-func (s *testSuite1) TestClusterIndexAnalyze(c *C) {
+func (s *testSerialSuite1) TestClusterIndexAnalyze(c *C) {
```

Comment:
We have decided to skip this test now https://github.com/pingcap/tidb/pull/20815.